docs/modules/examples/pages/knowledge-store.adoc

```diff
@@ -20,7 +20,7 @@ Create a knowledge store and use it to answer questions with graph RAG chains.
 +
 [source,bash]
 ----
-pip install ragstack-ai-langchain[knowledge-store] beautifulsoup4 markdownify python-dotenv
+pip install ragstack-ai-langchain ragstack-ai-knowledge-store beautifulsoup4 markdownify python-dotenv
```

Comment:
Why change ?

Reply:
pip install ragstack-ai-langchain[knowledge-store] doesn't work for me:
`pip install ragstack-ai-langchain[knowledge-store]
zsh: no matches found: ragstack-ai-langchain[knowledge-store]`

`pip install ragstack-ai-knowledge-store
Collecting ragstack-ai-knowledge-store
...
Successfully installed ragstack-ai-knowledge-store-0.0.5`

Reply:
This is a shell problem -- zsh is trying to do something with the `[knowledge-store]` (possibly treating it as a regular expression). If you change to `pip install ... "ragstack-ai-langchain[knowledge-store]" ...` (note the added quotes) it will prevent your shell from doing that and should work.